numpy/polynomial/chebyshev.py

```diff
@@ -2007,9 +2007,9 @@ class Chebyshev(ABCPolyBase):
     domain : (2,) array_like, optional
         Domain to use. The interval ``[domain[0], domain[1]]`` is mapped
         to the interval ``[window[0], window[1]]`` by shifting and scaling.
-        The default value is [-1, 1].
+        The default value is [-1., 1.].
     window : (2,) array_like, optional
-        Window, see `domain` for its use. The default value is [-1, 1].
+        Window, see `domain` for its use. The default value is [-1., 1.].
```

Comment:
These are unrelated docstring fixes, right?  (I don't mind, just checking)

Reply:
A tiny bit related: the window and domain type impacts the output type of the Polynomial evaluation.